test-files/cases/attribute/unifying/highway-2927-unmarked-ref-divided-road-1/Expected.osm

```diff
@@ -269,6 +269,36 @@
         <tag k="hoot:status" v="1"/>
         <tag k="hoot:id" v="-1"/>
     </node>
+    <way visible="true" id="-87" timestamp="2019-01-17T19:50:06Z" version="1">
+        <nd ref="-1"/>
+        <nd ref="-18"/>
+        <nd ref="-19"/>
+        <nd ref="-20"/>
+        <nd ref="-39"/>
+        <tag k="source" v="Yahoo;osm:line"/>
+        <tag k="way_area" v="-999999"/>
+        <tag k="license" v="This data is made available under the Open Database License: http://opendatacommons.org/licenses/odbl/1.0/."/>
+        <tag k="oneway" v="yes"/>
+        <tag k="tags" v="\&quot;source:datetime\&quot;=&gt;\&quot;2018-10-30T13:05:51Z\\"/>
+        <tag k="name" v="Av. Sur 4"/>
+        <tag k="lanes" v="3"/>
+        <tag k="cycleway" v="no"/>
+        <tag k="surface" v="asphalt"/>
+        <tag k="sidewalk" v="both"/>
+        <tag k="maxspeed" v="40"/>
+        <tag k="attribution" v="osm"/>
+        <tag k="source:ingest:datetime" v="2019-01-17T19:50:05.667Z;2019-01-17T19:50:05.631Z;2019-01-17T19:50:05.672Z"/>
+        <tag k="z_order" v="-999999"/>
+        <tag k="highway" v="secondary"/>
+        <tag k="source:datetime" v="2015-08-24T16:13:18.000Z;2019-01-17T19:50:06Z"/>
+        <tag k="foot" v="yes"/>
+        <tag k="uuid" v="{337b5b66-e2e1-4cb7-83e6-bad10ea5d108};{bd5d6e86-1289-412d-81b9-41087b55e1fd};{95ca7b93-cd09-4c19-a70c-b1ed653fb9ae}"/>
+        <tag k="bicycle" v="yes"/>
+        <tag k="osm_id" v="-999999"/>
+        <tag k="hoot:status" v="3"/>
+        <tag k="error:circular" v="15"/>
+        <tag k="hoot:id" v="-87"/>
+    </way>
```

Comment:
This one way is a joined way for all of the missing ways below and replaces part of the rearranged way below that gives better data.

Reply:
![screenshot from 2019-02-26 07-52-37](https://user-images.githubusercontent.com/303724/53413967-8ecca080-399b-11e9-8dca-8e9c7cd3b756.png)

This looks like another dogleg situation, similar to those in highway-2888.  The secondary geometry is much better than the ref there, unfortunately.